add-column.md

```diff
@@ -18,6 +18,112 @@ The user must have the `CREATE` [privilege](privileges.html) on the table.
+
 ## Examples
 
-## See Also
\ No newline at end of file
+### Add a Single Column
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN names STRING;
+~~~ 
+
+~~~ sql
+> SHOW COLUMNS FROM accounts;
+~~~ 
+
+~~~
++-----------+-------------------+-------+---------+-----------+
+|   Field   |       Type        | Null  | Default |  Indices  |
++-----------+-------------------+-------+---------+-----------+
+| id        | INT               | false | NULL    | {primary} |
+| balance   | DECIMAL           | true  | NULL    | {}        |
+| names     | STRING            | true  | NULL    | {}        |
++-----------+-------------------+-------+---------+-----------+
+~~~
+
+### Add Multiple Columns
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN location STRING, ADD COLUMN amount DECIMAL;
+~~~
+
+~~~ sql
+> SHOW COLUMNS FROM accounts;
+~~~ 
+
+~~~
++-----------+-------------------+-------+---------+-----------+
+|   Field   |       Type        | Null  | Default |  Indices  |
++-----------+-------------------+-------+---------+-----------+
+| id        | INT               | false | NULL    | {primary} |
+| balance   | DECIMAL           | true  | NULL    | {}        |
+| names     | STRING            | true  | NULL    | {}        |
+| location  | STRING            | true  | NULL    | {}        |
+| amount    | DECIMAL           | true  | NULL    | {}        |
++-----------+-------------------+-------+---------+-----------+
+
+~~~
+
+### Add a Non-Null Column with a Default Value
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN interest DECIMAL NOT NULL DEFAULT (DECIMAL '1.3');
+~~~ 
+
+~~~ sql
+> SHOW COLUMNS FROM accounts;
+~~~ 
+~~~
++-----------+-------------------+-------+---------------------------+-----------+
+|   Field   |       Type        | Null  |          Default          |  Indices  |
++-----------+-------------------+-------+---------------------------+-----------+
+| id        | INT               | false | NULL                      | {primary} |
+| balance   | DECIMAL           | true  | NULL                      | {}        |
+| names     | STRING            | true  | NULL                      | {}        |
+| location  | STRING            | true  | NULL                      | {}        |
+| amount    | DECIMAL           | true  | NULL                      | {}        |
+| interest  | DECIMAL           | false | ('1.3':::STRING::DECIMAL) | {}        |
++-----------+-------------------+-------+---------------------------+-----------+
+~~~
+
+### Add a Non-Null Column with Unique Values
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN cust_number DECIMAL UNIQUE NOT NULL;
+~~~ 
+ 
+### Add a Column with Collation
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN more_names STRING COLLATE en;
+~~~ 
+ 
+### Add a Column to a Column Family
+
+~~~ sql
+# Add a Column to a New Column Family
```

Comment:
Looking at the rendered page, I realize that comments in `sql` code blocks pick up weird syntax highlighting. So Instead of using comments in each of these, just precent the code blocks with h4 headers:

```
#### Assign a Column to a New Column Family
~~~ sql
xxx
~~~

#### Assign a Column to an Existing Column Family

~~~ sql
xxx
~~~

etc. 
```

Also, I think renaming the h3 and these h4 headers to use `Assign` instead of `Add` would more clearly evoke the focus on the examples.